src/ast.ls

```diff
@@ -3228,6 +3229,8 @@ UTILS =
       }
     }'''
 
+    array-from: 'Array.from || function(x){return slice$.call(x);}'
```

Comment:
Can the second part of "or" just be `slice$.call`? Or is there a reason for the wrapping function